sdk/ml/azure-ai-ml/tests/test_configs/automl_job/test_datasets/image_instance_segmentation/train/train_annotations.jsonl

```diff
@@ -0,0 +1,102 @@
+{"image_url": "azureml://subscriptions/102a16c3-37d3-48a8-9237-4c9b1e8e80e0/resourcegroups/risha-rg/workspaces/risha-ws-centraluseuap/datastores/workspaceblobstore/paths/LocalUpload/57e0ce1a06d180c7be40a8dc137a09c5/odFridgeObjectsMask/images/100.jpg", "image_details": {"format": "jpg", "width": 499, "height": 666}, "label": [{"label": "can", "bbox": "null", "isCrowd": 0, "polygon": [[0.5771543086172345, 0.6891891891891891, 0.5671342685370742, 0.6891891891891891, 0.5591182364729459, 0.6861861861861862, 0.3807615230460922, 0.5930930930930931, 0.3046092184368738, 0.5555555555555556, 0.29458917835671344, 0.545045045045045, 0.2905811623246493, 0.5345345345345346, 0.2745490981963928, 0.512012012012012, 0.27054108216432865, 0.496996996996997, 0.27054108216432865, 0.47897897897897895, 0.2845691382765531, 0.45345345345345345, 0.30861723446893785, 0.43243243243243246, 0.32665330661322645, 0.42342342342342343, 0.35671342685370744, 0.4159159159159159, 0.4188376753507014, 0.4174174174174174, 0.6352705410821643, 0.493993993993994, 0.6833667334669339, 0.5075075075075075, 0.7014028056112225, 0.5180180180180181, 0.7094188376753507, 0.5285285285285285, 0.7134268537074149, 0.545045045045045, 0.7194388777555111, 0.5540540540540541, 0.7194388777555111, 0.5795795795795796, 0.7134268537074149, 0.5975975975975976, 0.6973947895791583, 0.6216216216216216, 0.6953907815631263, 0.6291291291291291, 0.6312625250501002, 0.6786786786786787, 0.6192384769539078, 0.6831831831831832, 0.5951903807615231, 0.6831831831831832, 0.5771543086172345, 0.6891891891891891]]}]}
```

Comment:
is there a way to create jsonl files on the fly and not have it created before?

Reply:
That is what we had earlier, and simplification package is used to create the mask... Since this is a test and dataset is fixed, we don't need to create mask every time... we can just keep updating the path to images to remove the simplification package dependency... 
Ideally, we should try and find a replacement for simplification package, since we have seen this happening before as well (simplification not being supported for some python versions), but for now that is not a priority afaik...